examples/grd/S1A_EW_GRDM_1SDH_20221130T014342_20221130T014446_046117_058549_BB15/S1A_EW_GRDM_1SDH_20221130T014342_20221130T014446_046117_058549_BB15.json

```diff
@@ -0,0 +1,228 @@
+      "roles": [
+        "metadata"
+      ]
+    },
+    "schema-noise-hh": {
+      "href": "../../../tests/data-files/grd/S1A_EW_GRDM_1SDH_20221130T014342_20221130T014446_046117_058549_BB15/annotation/calibration/noise-ew-hh.xml",
+      "type": "application/xml",
+      "title": "Noise Schema",
+      "description": "Estimated thermal noise look-up tables",
+      "roles": [
+        "metadata"
+      ]
+    },
+    "schema-noise-hv": {
+      "href": "../../../tests/data-files/grd/S1A_EW_GRDM_1SDH_20221130T014342_20221130T014446_046117_058549_BB15/annotation/calibration/noise-ew-hv.xml",
+      "type": "application/xml",
+      "title": "Noise Schema",
+      "description": "Estimated thermal noise look-up tables",
+      "roles": [
+        "metadata"
+      ]
+    },
+    "thumbnail": {
+      "href": "../../../tests/data-files/grd/S1A_EW_GRDM_1SDH_20221130T014342_20221130T014446_046117_058549_BB15/preview/quick-look.png",
+      "type": "image/png",
+      "title": "Preview Image",
+      "description": "An averaged, decimated preview image in PNG format. Single polarization products are represented with a grey scale image. Dual polarization products are represented by a single composite colour image in RGB with the red channel (R) representing the  co-polarization VV or HH), the green channel (G) represents the cross-polarization (VH or HV) and the blue channel (B) represents the ratio of the cross an co-polarizations.",
+      "roles": [
+        "thumbnail"
+      ]
+    },
+    "hh": {
+      "href": "../../../tests/data-files/grd/S1A_EW_GRDM_1SDH_20221130T014342_20221130T014446_046117_058549_BB15/measurement/ew-hh.tiff",
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "title": "HH",
+      "description": "Actual SAR data that have been processed into an image",
+      "eo:bands": [
+        {
+          "name": "HH",
+          "description": "HH band: horizontal transmit and horizontal receive"
+        }
+      ],
+      "roles": [
+        "data"
+      ]
+    },
+    "hv": {
+      "href": "../../../tests/data-files/grd/S1A_EW_GRDM_1SDH_20221130T014342_20221130T014446_046117_058549_BB15/measurement/ew-hv.tiff",
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "title": "HV",
+      "description": "Actual SAR data that have been processed into an image",
+      "eo:bands": [
+        {
+          "name": "HV",
+          "description": "HV band: horizontal transmit and vertical receive"
+        }
+      ],
+      "roles": [
+        "data"
+      ]
+    }
+  },
+  "bbox": [
+    75.348396,
+    76.879097,
+    102.789734,
+    81.972343
+  ],
+  "stac_extensions": [
+    "https://stac-extensions.github.io/sar/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/sat/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/eo/v1.0.0/schema.json"
+  ],
+  "collection": "sentinel1-grd"
+}
```

Comment:
Did the linter complain about no newline at the end of the file? On cop-dem which is the older linting method I had to exclude the examples folder from linting.

Reply:
It didn't raise anything while running linting in the container 🤷🏼‍♂️ I can go back in and add newlines. My IDE settings just didn't automatically add one because these generated files didn't need to be saved